backend/tool_instance_v2/urls.py

```diff
@@ -42,5 +42,10 @@
             name="tool_instance_reorder",
         ),
         path("tool/", views.get_tool_list, name="tool_list"),
+        path(
+            "tool/prompt-studio/",
```

Comment:
NIT: Its not clear from the API name that we're trying to obtain a count of the entity. Change it in the FE as well
```suggestion
            "tool/prompt-studio/count/",
```